### PR TITLE
Fancy new tab control

### PR DESCRIPTION
**Pull request by ticky**

### CODE REVIEW

File: app/components/shared/TabControl/Tab.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  className: 'px3 pt2 pb1 border-bottom block text-decoration-none black',
+  activeClassName: ACTIVE_CLASS_NAME
+})`
+  border-bottom-width: 4px;
+  border-color: transparent;
+
+  &.${ACTIVE_CLASS_NAME} {
+    border-color: ${cssVariables['--lime']};
+  }
+`;
```
Comment (ticky) [inline app/components/shared/TabControl/Tab.js]: some comments I made on this stuff while I was querying Tim’s opinion on it:

> it requires enough stuff basscss itself can’t do alone that I’ve had to use it with styled-components, but also enough stuff styled-components and basscss can’t do alone that I’ve had to implement a small custom component

(this is why I’m not just using `border-lime` in `activeClassName`)

> it’s probably mostly good but it feels inelegant
> it’s mostly complicated by react-router not having good utilities for querying or reacting to state outside of `Router.Link`

File: app/components/team/Show.js [hunk 2]
```diff
@@ -58,13 +67,60 @@ class TeamShow extends React.Component {
...
+          {this.renderTabs()}
+
+          {this.props.children}
         </div>
       </DocumentTitle>
     );
   }
 
+  renderTabs() {
+    const tabContent = permissions(this.props.team.permissions).collect(
```
Comment (keithpitt) [inline app/components/team/Show.js]: Nice, I'm glad it worked out here as well :)

File: app/components/shared/TabControl/Tab.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const TabButton = styled(Link).attrs({
+  className: 'px3 pt2 pb1 border-bottom block text-decoration-none black',
+  activeClassName: ACTIVE_CLASS_NAME
+})`
+  border-bottom-width: 4px;
+  border-color: transparent;
+
+  &.${ACTIVE_CLASS_NAME} {
+    border-color: ${cssVariables['--lime']};
```
Comment (keithpitt) [inline app/components/shared/TabControl/Tab.js]: `cssVariables` very nice

Comment (ticky) [inline app/components/shared/TabControl/Tab.js]: Glad you like it! 😃

**Pull request merged**